bucket/nu.json

```diff
@@ -0,0 +1,18 @@
+{
+    "homepage": "https://www.nushell.sh",
+    "version": "0.3.0",
+    "license": "MIT",
+    "description": "A modern shell written in Rust.",
+    "url": "https://github.com/nushell/nushell/releases/download/0.3.0/nu_0_3_0_windows.zip",
+    "hash": "73f5ac08c3c3fe1f81ec45bcae95985068f859d3aceb1267e88b924317e227f7",
+    "extract_dir": "nushell_0_3_0",
+    "env_add_path": ".",
+    "checkver": {
+        "github": "https://github.com/nushell/nushell",
+        "regex": "/tag/([d.]+)"
```

Comment:
??
Why you need special regex for this?

Reply:
Wo, yes, normal regex works.